README.md

```diff
@@ -230,15 +230,14 @@ if err := stats.StopForcedCollection(myView1); err != nil {
 ```
 
 ### Recording measurements
-Recording usage can only be performed against already registered measure and and their registered views. Measurements are implicitly tagged with the tags in the context:
+Recording usage can only be performed against already registered measure
+and their registered views. Measurements are implicitly tagged with the
+tags in the context:
 
 ```go
-// mi is a *MeasureInt64 and v is an int64 .
-stats.RecordInt64(ctx, mi, v)
-// mf is a *RecordFloat64 and v is an float64 .
-stats.RecordFloat64(ctx, mf, v)
-// multiple measurements can be performed at once.
-stats.Record(ctx, mi.Is(4), mf.Is(10.5))
+mi.Record(ctx, 1)
+mf.Record(ctx, 5.6)
+stats.Record(ctx, mi.M(4), mf.M(10.5))
```

Comment:
What are the use cases for the batch versus the single record modes here?  The sample doesn't illuminate why the user would want to use one or the other.

Reply:
There are tons of cases where you want to record a set of measurements at certain points. For example, a response is returned from an outgoing RPC. The size, whether the RPC has been successful, latency, and similar measurements are all made available at the same time. Batch is useful for those cases, but I agree that we need to reconsider the APIs. I am going to add a TODO for it for now.

Reply:
SGTM.  My recommendation is (when the API is stable) to pick an example that illustrates when to use each.